src/Execute/executeQuickInput.ts

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {exec} from 'child_process';
+import {appendFileSync} from 'fs';
+import * as vscode from 'vscode';
+
+import {Backend} from '../Backend/API';
+import {globalBackendMap} from '../Backend/Backend';
+import {MultiStepInput} from '../Utils/MultiStepInput';
+
+export async function runInferenceQuickInput(context: vscode.ExtensionContext) {
+  interface State {
+    backend: Backend;
+    modelPath: vscode.Uri;
+    inputSpec: string;
+    selectedItem: vscode.QuickPickItem;
+    error: string|undefined;
+  }
+
+  async function collectInputs() {
+    const state = {error: undefined} as Partial<State>;
+    await MultiStepInput.run(input => pickBackend(input, state));
+    return state as State;
+  }
+
+  async function pickBackend(input: MultiStepInput, state: Partial<State>) {
+    const backends: vscode.QuickPickItem[] = Object.keys(globalBackendMap).map(label => ({label}));
+    state.selectedItem = await input.showQuickPick({
+      title: 'Choose Executor Toolchain',
+      step: 1,
+      totalSteps: 3,
+      placeholder: 'Select a Backend',
+      items: backends,
+      shouldResume: shouldResume
+    });
+    state.backend = globalBackendMap[state.selectedItem.label];
+    return (input: MultiStepInput) => selectInputModel(input, state);
+  }
+
+  // TODO: Use quickPick window with fast grep child process
+  async function selectInputModel(input: MultiStepInput, state: Partial<State>) {
+    if (state.backend === undefined) {
+      state.error = 'Backend to infer is not chosen. Please check once again.';
+      return;
+    }
+    if (state.backend.executor() == undefined) {
+      state.error = 'Backend executor is not set yet. Please check once again.';
+      return;
+    }
```

Comment:
I've added error handling param to the state. The lack of information to infer a model, which can be `backend`, `executor`, `modelPath`, options for executor command, will add a error message.

/cc @YongseopKim